app/grandchallenge/reader_studies/models.py

```diff
@@ -193,6 +219,12 @@ class ReaderStudy(
     a set of questions on a set of images (cases).
     """
 
+    title = models.CharField(
+        _("title"),
+        max_length=255,
+        validators=[CleanedHtmlValidator(no_tags=True)],
+    )
+
```

Comment:
Also a _slight_ question about this one: Is this ok? This _redefines_ the title-field from `TitleSlugDescriptionModel` to add the validator. I find it a bit messy. Is there a better way?

Reply:
Actually, maybe there is... what about this construct:

```.py
def init_from_serialized(cls_args_kwargs, **overload_kwargs):
    cls, args, kwargs = cls_args_kwargs
    return cls(*args, **{**kwargs, overload_kwargs})

title = init_from_serialized(
    TitleSlugDescriptionModel.title.deconstruct(),
    validators=[validators=[CleanedHtmlValidator(no_tags=True)]]
)
```

Or is that getting too opaque with too much meta programming?

Reply:
Yes too meta, we pretty much never change the title base class anyway.